ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchaserInfoHybridAdditionsTests.swift

```diff
@@ -8,95 +8,97 @@
 
 import Quick
 import Nimble
-import PurchasesCoreSwift
-import Purchases
+@testable import PurchasesHybridCommon
+@testable import RevenueCat
 
-class PurchaserInfoHybridAdditionsTests: QuickSpec {
+class CustomerInfoHybridAdditionsTests: QuickSpec {
 
     override func spec() {
         describe("rc_dictionary") {
             context("managementURL") {
                 it("contains the management url when it exists") {
-                    let purchaserInfo = PartialMockPurchaserInfo()
+                    let mockCustomerInfo = try! CustomerInfo(data: [
```

Comment:
I think `Quick` allows throwing exceptions in `it`, no? So this can be `try`?

Reply:
nice! I didn't know that